src/database/config.js

```diff
@@ -0,0 +1,12 @@
+import { Pool } from 'pg';
+
+import dotenv from 'dotenv';
+
+dotenv.config();
+const connetionstring = process.env.DATABASE_URL;
```

Comment:
It's important to have a separate database for testing, development and production phases. We also need to follow the naming convention when creating variables 
```suggestion
const dbConnection = () => {
  const { NODE_ENV } = process.env;
  let dbURL;
  if (NODE_ENV === 'test') {
    dbURL = process.env.DATABASE_TEST_URL;
  } else if (NODE_ENV === 'development') {
    dbURL = process.env.DATABASE_DEV_URL;
  } else {
    dbURL = process.env.DATABASE_URL;
  }
  return dbURL;
};
const connectionString = dbConnection();
```

Reply:
Alright